adcm_aio_client/core/objects/cm.py

```diff
@@ -110,3 +116,56 @@ def get_own_path(self: Self) -> Endpoint:
 
 class ComponentsNode(NonPaginatedChildAccessor[Service, Component, None]):
     class_type = Component
+
+
+class ConfigsNode(PaginatedChildAccessor): ...
+
+
+class HostProvidersNode(PaginatedChildAccessor): ...
+
+
+class Host(InteractiveChildObject[Cluster | None]):
```

Comment:
`Cluster` is not a parent of `Host` (I see that you've tried to cover case with "Host of Cluster", but IMO it's too early for such stretch)

Reply:
changed